src/voice-player.html

```diff
@@ -1,14 +1,24 @@
-<!-- Import Polymer -->
 <link rel="import" href="../bower_components/polymer/polymer.html">
```

Comment:
I would suggest switching over to polyserve to get rid of the bower_components stuff. I showed a demo in one of the polycasts https://www.youtube.com/watch?v=p7Q1mQtFGM8&list=PLOU2XLYxmsII5c3Mgw6fNYCzaWrsM3sMN&index=2